app/inputs/fancy_color_input.rb

```diff
@@ -0,0 +1,12 @@
+class FancyColorInput < SimpleForm::Inputs::ColorInput
+  def input(wrapper_options)
+    merged_input_options = merge_wrapper_options(input_html_options, wrapper_options)
+
+    "<div class='form-control fancy-color-container row'><div class='col fancy-color-picker-col'>" +
+      @builder.color_field(attribute_name, merged_input_options) +
+      "</div><div class='col'>" + 
+      @builder.input_field(attribute_name, merged_input_options) +
+      "</div>" + 
+    "</div>".html_safe
```

Comment:
This level of HTML string manipulation is generally frowned upon - [`tag`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TagHelper.html#method-i-tag) and [`content_tag`](https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag) are things.